dashboard-ui/src/components/TextBasedScenarios/TextBasedScenariosPage.jsx

```diff
@@ -201,6 +200,32 @@ class TextBasedScenariosPage extends Component {
         return page ? page.questions.map(question => question.name) : [];
     };
 
+    /*mapAnswers = (scenario) => {
+        // maps the user's answer to the correct naming convention for ADEPT choice id
+        const scenarioConfig = scenarioMappings[scenario.title.replace(' Scenario', '')];
+        Object.entries(scenario).forEach(field => {
+            if (!field[1].questions) { return; }
+            Object.entries(field[1].questions).forEach(question => {
+                if (!question[1].probe) { return; }
+                const page = scenarioConfig.pages.find((page) => page.name === field[0]);
+                const pageQuestion = page.elements.find((element) => element.name === question[0]);
+                const indexOfAnswer = pageQuestion.choices.indexOf(question[1].response);
+                let choice;
+                if (scenario.title.includes("Adept")) {
+                    if (indexOfAnswer >= 0) {
+                        choice = question[1].probe + '.';
+                        choice += String.fromCharCode(65 + indexOfAnswer);
+                    } else {
+                        console.err("Error mapping user selection to choice ID");
+                    }
+                } else {
+                    choice = "choice-"+indexOfAnswer;
+                }
+                question[1].choice = choice;
+            })
+        })
+    }*/
+
```

Comment:
Is this intentionally left here?

Reply:
Well ideally I'd like to mark the choice id to the documents before they get uploaded to mongo instead of having to write the script so that is the purpose of this being here. This will get expanded on so I think I'll just leave it for now

Reply:
Great, I just wanted to make sure :)